ffig/filters/capi_filter.py

```diff
@@ -78,33 +80,37 @@ def to_go_method_name(m):
     return m.capitalize()
 
 
-def go_object(v, t):
+def go_object(a):
     '''
-    In analogy to the `c_object` filter below, this returns `v.ptr` for objects
-    of class type, and `v` for everything else.
+    In analogy to the `c_object` filter below, this returns `a.ptr` for objects
+    of class type, and `a` for everything else.
```

Comment:
This comment is wrong. It returns `'{}.ptr'.format(a.name)` for objects of class type, and 'a.name` for everything else.